src/components/ui/InputText/InputText.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import type { InputTextProps } from '.'
+import InputText from '.'
+
+export default {
+  component: InputText,
+  title: 'Atoms/InputText',
+  argTypes: {
+    inputRef: {
+      table: {
+        disable: true,
+      },
+    },
+    id: {
+      type: { name: 'string', required: true },
+    },
+  },
+}
+
+const Template = ({ ...args }: InputTextProps) => (
+  <div style={{ width: 400 }}>
+    <InputText {...args} />
+  </div>
+)
+
+export const Default = Template.bind({})
+export const HasError = Template.bind({})
+export const Actionable = Template.bind({})
+export const Disabled = Template.bind({})
+
+Default.args = {
+  id: 'default-input-text',
+  label: 'Email',
+  errorMessage: 'Please, add a valid email',
+  disabled: false,
+}
+
+HasError.args = {
+  id: 'error-input-text',
+  label: 'Email',
+  value: 'invalid@mail',
+  errorMessage: 'Please, add a valid email',
+  disabled: false,
+}
+
+Actionable.args = {
+  id: 'actionable-input-text',
+  label: 'Postal Code',
+  actionable: true,
+  errorMessage: 'Invalid Postal Code',
+  disabled: false,
```

Comment:
Do we have to consider that actionable can be disabled?
![CleanShot 2022-05-03 at 21 28 31](https://user-images.githubusercontent.com/381395/166608144-08debec3-8b57-43a8-82cf-b69ae1c09813.gif)

Reply:
I was wondering the same thing! Maybe we could prevent this scenario 🤔 
@renatamottam How about hiding the action button when disabled?